contracts/eos/BancorConverter/BancorConverter.hpp

```diff
@@ -408,7 +435,7 @@ class [[eosio::contract]] BancorConverter : public contract { /*! \endcond */
         static uint128_t _by_cnvrt( asset balance, symbol_code converter_currency_code ) {
            return ( uint128_t{ balance.symbol.code().raw() } << 64 ) | converter_currency_code.raw();
         }
-        constexpr static double MAX_RATIO = 1000000.0;
+        constexpr static double MAX_WEIGHT = 1000000.0;
```

Comment:
can simply use PPM_RESOLUTION for both here

Reply:
Renamed ✅ 

https://github.com/bancorprotocol/contracts_eos/pull/72/commits/06c03d391c401b3089b97b7031fbac62490a8c8b